README.md

```
@@ -1,3 +1,61 @@
 # java-janggi
 
-장기 미션 저장소
+콘솔 입출력으로 진행 가능한 장기 게임을 자바로 구현했습니다.<br>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: br

(MD033, no-inline-html)